<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-R44QD4T" failures="4" tests="5" name="Test" time="44.987" errors="0" timestamp="29 Feb 2020 20:25:15 GMT">
  <testcase classname="com.tests.Excercise1Test" name="excercise1" time="31.038"/>
  <testcase classname="com.tests.Excercise2Test" name="excercise2" time="0.002">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.tests.Excercise2Test.excercise2(Excercise2Test.java:17)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- excercise2 -->
  <testcase classname="com.tests.Excercise3Test" name="excercise3" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.tests.Excercise3Test.excercise3(Excercise3Test.java:15)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- excercise3 -->
  <testcase classname="com.tests.Excercise4Test" name="excercise4" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.tests.Excercise4Test.excercise4(Excercise4Test.java:12)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- excercise4 -->
  <testcase classname="com.tests.Excercise5Test" name="excercise5" time="0.0">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.tests.Excercise5Test.excercise5(Excercise5Test.java:14)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- excercise5 -->
</testsuite> <!-- Test -->
